src/AblyRest.php

```diff
@@ -1,7 +1,8 @@
 <?php
+namespace Ably;
 
-require_once dirname(__FILE__) . '/AuthMethod.php';
-require_once dirname(__FILE__) . '/Channel.php';
+use Ably\Exceptions\AblyException;
+use Ably\Exceptions\AblyRequestException;
```

Comment:
So I suppose these classes are now automatically loaded. Is that a default PHP behavior?